indico/htdocs/js/indico/modules/timetable/timetable/Draw.js

```diff
@@ -551,46 +548,123 @@ type("TimetableBlockWholeDayBase", ["TimetableBlockBase"],
+    var params = {
+       'confId': self.eventData.conferenceId[0],
+       'timetable_entry_id': entryId
+    };
+    if(editable) {
+        params['editable'] = true;
+    }
+    var balloonId = '#qtip-' + entryId.toString();
+
+    $(timetableBlock).qbubble({
+       id: entryId.toString(),
+       content: {
+           text: function(event, api) {
+               $.ajax({
+                   url: build_url(Indico.Urls.Timetable.entries.balloon, params)
+               })
+               .then(function(content) {
+                   // Set the tooltip content upon successful retrieval
+                   api.set('content.text', content.html);
+                   var $content = api.elements.content;
+                   if(editable) {
+                       $duration = $content.find('.js-edit-time');
+                       $duration.ajaxqbubble({
+                           url: build_url(Indico.Urls.Timetable.entries.editTime, params),
+                           qBubbleOptions: {
+                               style: {
+                                   classes: 'balloon-time-qtip'
+                               },
+                               position: {
+                                   at: 'top center',
+                                   my: 'bottom center',
+                                   target: $(timetableBlock)
+                               }
+                           },
+                           onClose: function(data) {
+                               if (data.entries) {
+                                   self.managementActions._addEntries(data.entries)
+                               }
+                           }
+                       });
+
+
+                       $content.find('.js-delete').on('click', function() {
+                           self.managementActions.deleteEntry(self.eventData);
+                       });
+
+                       $content.find('.js-edit').on('click', function() {
+                           ajaxDialog({
+                               trigger: this,
+                               url: build_url(Indico.Urls.Timetable.entries.edit, params),
+                               title: $(this).data('title'),
+                               onClose: function(data) {
+                                   if (data && data.entries) {
+                                       self.managementActions._addEntries(data.entries)
+                                   }
+                               }
+                           });
+                       });
+                   }
+                   $content.find('.push-balloon-back').each(function() {
+                       $(this).on('click', function() {
+                           $(balloonId).addClass('push-back');
+                       });
+                   });
+                   // Change the target of the qTip position in order to open it at the mouse position
+                   api.options.position.target = 'mouse';
```

Comment:
No, that's the hack. You initialize it in the X Y position of the mouse in order for it to open at the mouse place, and then I change it to `mouse` so that every time it opens again it will follow the mouse and not the initial position.